Sources/Swift/Tools/RedactRegion.swift

```diff
@@ -0,0 +1,48 @@
+#if canImport(UIKit)
+import Foundation
+import UIKit
+
+struct RedactRegion {
+    let rect: CGRect
+    let color: UIColor?
+    
+    init(rect: CGRect, color: UIColor? = nil) {
```

Comment:
`m`: What made you choose to make the color nullable? Only the tests omit this parameter. The source code always passes in the color.

Reply:
Color can be nil, see `color` function from `SentryViewPhotographer`

```swift
    private func color(for view: UIView) -> UIColor? {
         return (view as? UILabel)?.textColor
     }
```